test/iron-request.html

```diff
@@ -136,6 +151,41 @@
           });
         });
 
+        test('setting jsonPrefix correctly strips it from the response', function () {
+          var options = {
+            url: '/responds_to_get_with_prefixed_json',
+            handleAs: 'json',
+            jsonPrefix: '])}while(1);</x>'
+          };
+
+          request.send(options);
+          expect(server.requests.length).to.be.equal(1);
+          expect(server.requests[0].requestHeaders['accept']).to.be.equal(
+              'application/json');
+          server.respond();
+
+          return request.completes.then(function() {
+            expect(request.response).to.be.an('object');
+          });
+        });
+
+        test('json prefix is correctly stripped from the response with header', function () {
+          var options = {
+            url: '/responds_to_get_with_prefixed_json_and_header',
+            handleAs: 'json'
+          };
+
+          request.send(options);
+          expect(server.requests.length).to.be.equal(1);
+          expect(server.requests[0].requestHeaders['accept']).to.be.equal(
+              'application/json');
+          server.respond();
+
+          return request.completes.then(function() {
+            expect(request.response).to.be.an('object');
```

Comment:
Here and in the test above, asserting that something is an Object doesn't tell you much. The object could be an instance of Error, or an empty object default value, etc.

I'd recommend something like:

```javascript
    expect(request.response).to.deep.eq({success: true});
```